lib/taskana-core/src/main/java/pro/taskana/TaskanaConfiguration.java

```diff
@@ -1135,6 +1135,15 @@ public TaskanaConfiguration build() {
 
     // endregion
 
+    private void addMasterDomain() {
+      // Master Domain is treat as empty string
+      // it must be always added to the configuration
+      // add the master domain always at the end of the list
+      if (!this.domains.contains("")) {
+        this.domains.add("");
```

Comment:
Please use SharedConstants#MASTER_DOMAIN, so that we can change the name of the master domain with ease :)

Reply:
Do this for all usages please :)

Reply:
Done